README.md

```diff
@@ -15,6 +15,16 @@ Run the development server:
 ```bash
 pnpm dev
 ```
+### Setup Clerk Authentication
+- SignUP with [Clerk](https://clerk.com)
+- Create Application
+- Give It a Name (Optional: Xplorer Test)
+- Leave Email and Gmail On
+- Create Application
+- Copy secrets on Step 2
+- Save it to .env.local
+
+> 💡 Any changes done on Your Clerk Account will not be reflected on Xplorer Production Website.
```

Comment:
Why is this even on readme ? Why someone would need this ? Any specific reason for adding ?